cfpb/cfgov-refresh
# FCM spacing updates

Removes flush-sides class from FCM, and top margin when FCM is first item in section. For more info, see hubcap issue 89

## Additions

- `u-mt0` class when FCM is first item in section. (`block__flush-top` class does not work here because it removes the top border)

## Removals

- `block__flush-sides` class from FCM

## Testing

1. Check out branch
2. Navigate to a page with an FCM [locally](http://localhost:8000/consumer-tools/bank-accounts/) and [on cf.gov](https://www.consumerfinance.gov/consumer-tools/bank-accounts/) and verify the changes to the FCM spacing

## Screenshots

### Before

<img width="1234" alt="screen shot 2018-06-07 at 9 09 30 am" src="https://user-images.githubusercontent.com/778171/41112020-95ca1fc2-6a32-11e8-83d0-170ccc232c8d.png">


### After

<img width="1267" alt="screen shot 2018-06-07 at 9 09 48 am" src="https://user-images.githubusercontent.com/778171/41112026-9a8d3120-6a32-11e8-8a7a-a9bfc08398da.png">




## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
